.github/workflows/build.yaml

```diff
@@ -33,10 +35,9 @@ jobs:
       run: |
         mypy daktari
     - name: test
+      working-directory: daktari
       run: |
-        sudo apt-get update
-        sudo apt-get install -y xclip xorg
-        python -m unittest discover
+        python -m pytest
```

Comment:
oh nice - it's been a few years, but I have a mental note that pytest is just way nicer, maybe at some point we should look to port our tests to take full advantage!

Reply:
yeah, the output is a lot better for starters :smile: